ambry-api/src/main/java/com/github/ambry/account/ContainerBuilder.java

```diff
@@ -311,6 +316,26 @@ public ContainerBuilder setSnapshotVersion(int snapshotVersion) {
     return this;
   }
 
+  /**
+   * Sets the cacheTtlInSecond of the {@link Container} to build.
+   * @param cacheTtlInSecond The value to set.
+   * @return This builder.
+   */
+  public ContainerBuilder setCacheTtlInSecond(Long cacheTtlInSecond) {
+    this.cacheTtlInSecond = cacheTtlInSecond;
```

Comment:
Probably this can only been set when Cacheable is true?

Reply:
I want to make this builder as simple as possible. When we use these values, we would check if the container is cacheable or not first.

Reply:
nit: probably the java doc can emphasize that in case someone don't aware of it.